pydis_site/apps/api/viewsets.py

```diff
@@ -192,7 +192,7 @@ class InfractionViewSet(CreateModelMixin, RetrieveModelMixin, ListModelMixin, Ge
     ### POST /bot/infractions
     Create a new infraction and return the created infraction.
     Only `actor`, `type`, and `user` are required.
-    The `actor` and `user` must be users known by the site.
+    The `actor` and `user` must be users known by the pydis_site.
```

Comment:
sed isn't gonna cut it, you'll have to undo these

Reply:
Haha, that's great. I did manually go through PyCharm's refactoring step, guess I missed some of them.